map/traffic_manager.cpp

```diff
@@ -344,8 +353,10 @@ void TrafficManager::CheckCacheSize()
       auto const it = m_mwmCache.find(mwmId);
       if (it->second.m_isLoaded)
       {
-        m_currentCacheSizeBytes -= it->second.m_dataSize;
+        // Note. It's necessary to multiply by two because routing and rendering use separate caches.
+        m_currentCacheSizeBytes -= 2 * it->second.m_dataSize;
```

Comment:
Why ```2 * ``` is here? You already multiplied by two in OnTrafficDataResponse, and saved result in m_dataSize.